Boolean.CSharp.Main/Core.cs

```diff
@@ -1,14 +1,123 @@
-﻿using System;
+﻿using Newtonsoft.Json;
+using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Text;
+using System.Transactions;
 
 
 namespace Boolean.CSharp.Main
 {
-    public class Core
+    public enum accountType
     {
+        current,
+        savings
+    }
+    public class Customer
+    {
+        private List<Account> accounts = new List<Account>();
+        public List<Account> Accounts { get { return accounts; } }
+        public Account createAccount(int accountNr, accountType type)
+        {
+            if(type == accountType.current)
+            {
+                CurrentAccount account = new CurrentAccount(accountNr);
+                accounts.Add(account);
+                return account;
+            }
+            else
+            {
+                SavingsAccount account = new SavingsAccount(accountNr);
+                accounts.Add(account);
+                return account;
+            }
+        }
+    }
+    public abstract class Account
+    {
+        private int _accountNr;
+        private float _balance = 0;
+        private List<Transaction> transaction = new List<Transaction>();
```

Comment:
variable name should be `transactions` since it represents a collection of things